api/src/pcapi/models/feature.py

```diff
@@ -208,6 +208,21 @@ def nameKey(self) -> str:
 if settings.IS_PROD or settings.IS_STAGING:
     FEATURES_DISABLED_BY_DEFAULT += (FeatureToggle.WIP_ENABLE_OFFER_CREATION_API_V1,)
 
+if settings.IS_TESTING:
+    testing_features_disabled_by_default = list(FEATURES_DISABLED_BY_DEFAULT)
+    features_to_enable = [
+        FeatureToggle.ENABLE_CHARLIE_BOOKINGS_API,
+        FeatureToggle.WIP_CATEGORY_SELECTION,
+        FeatureToggle.WIP_ENABLE_EVENTS_WITH_TICKETS_FOR_PUBLIC_API,
+        FeatureToggle.WIP_ENABLE_OFFER_CREATION_API_V1,
+        FeatureToggle.WIP_ENABLE_NEW_USER_OFFERER_LINK,
+    ]
+    for feature_to_enable in features_to_enable:
+        if feature_to_enable in testing_features_disabled_by_default:
+            testing_features_disabled_by_default.remove(feature_to_enable)
```

Comment:
```suggestion
    testing_features_disabled_by_default = set(FEATURES_DISABLED_BY_DEFAULT)
    features_to_enable = {
        FeatureToggle.ENABLE_CHARLIE_BOOKINGS_API,
        FeatureToggle.WIP_CATEGORY_SELECTION,
        FeatureToggle.WIP_ENABLE_EVENTS_WITH_TICKETS_FOR_PUBLIC_API,
        FeatureToggle.WIP_ENABLE_OFFER_CREATION_API_V1,
        FeatureToggle.WIP_ENABLE_NEW_USER_OFFERER_LINK,
    }
    testing_features_disabled_by_default - features_to_enable
```
(optionnel) On pourrait aussi écrire une compréhension de liste, mais j'ai une certaine affection pour les _sets_.